src/test/java/teammates/test/cases/BaseComponentTestCase.java

```diff
@@ -31,6 +31,7 @@
 
     protected static final GaeSimulation gaeSimulation = GaeSimulation.inst();
     protected static final BackDoorLogic backDoorLogic = new BackDoorLogic();
+    protected DataBundle typicalBundle = getTypicalDataBundle();
```

Comment:
Also, the method body of `prepareTestData` in `BaseActionTest` can do without the first line.